src/components/PayloadDetailsCard.js

```diff
@@ -195,23 +212,26 @@ function ExtrinsicPart({ label, fallback, prefix, value }) {
   }
 
   const renderMethodDetails = () => {
-    return (
-      argNameValue && sectionMethod && (
-        <View style={{ display: 'flex', flexDirection: 'column' }}>
-          <Text style={styles.secondaryText}>
-            You are calling <Text style={styles.secondaryText}>{sectionMethod}</Text> with the following arguments:
-          </Text>
+    if (formattedCallArgs) {
+      return Object.entries(formattedCallArgs).reverse().map((entry, index) => {
+        const sectionMethod = entry[0];
+        const paramArgs = entry[1];
+  
+        return (
+          <View key={index} style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start', justifyContent: 'flex-start', paddingLeft  : 5, width: '100%' }}>
+            <Text>Call <Text style={styles.secondaryText}>{sectionMethod}</Text> with the following arguments:</Text>
             {
-              Object.entries(argNameValue).map(([key, value]) => { return (
-                <View key={key} style={{ display: 'flex', flexDirection: 'row', flexWrap: 'wrap', padding: 5, alignItems: 'flex-start' }}>
-                  <Text style={{...styles.subLabel, flex: 1}}>{key}: </Text>
-                  <Text style={{...styles.secondaryText, flex: 3}}>{value}</Text>
-                </View>
-              )})
+              paramArgs.map(([param, arg]) => (
+                <React.Fragment key={param}>
+                  <Text style={{...styles.subLabel, flex: 1}}>{param}: </Text>
+                  <Text style={{...styles.secondaryText, flex: 3}}>{arg || undefined}</Text>
```

Comment:
If arg is `undefined`, then we return `undefined`, which is still not a string here.